other/materials_designer/create_slab.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Remove Materials are loaded according to&nbsp;<a href="https://github.com/Exabyte-io/api-examples/raw/b4c0ab1decf1490bc3a7b1297e389500fca6a166/README.ipynb" target="_blank">README</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>